objects:
- approach_leave:
    right_j0: -0.12352734375
    right_j1: -0.8837490234375
    right_j2: 1.3232265625
    right_j3: 1.89214453125
    right_j4: 2.3434736328125
    right_j5: -1.2021943359375
    right_j6: 3.1951748046875
  approach_pick:
    right_j0: -0.757482421875
    right_j1: -1.2157841796875
    right_j2: 0.43545703125
    right_j3: 2.06341796875
    right_j4: 2.905306640625
    right_j5: -0.7395185546875
    right_j6: 3.191482421875
  leave:
    right_j0: -0.16825390625
    right_j1: -0.5264384765625
    right_j2: 1.36505859375
    right_j3: 1.90353125
    right_j4: 2.0191640625
    right_j5: -1.25596875
    right_j6: 3.32755078125
  leave_pick:
    right_j0: -0.757482421875
    right_j1: -1.2157841796875
    right_j2: 0.43545703125
    right_j3: 2.06341796875
    right_j4: 2.905306640625
    right_j5: -0.7395185546875
    right_j6: 3.191482421875
  pick:
    right_j0: -0.8732744140625
    right_j1: -0.9080615234375
    right_j2: 0.43574609375
    right_j3: 2.157234375
    right_j4: 2.643794921875
    right_j5: -0.3966806640625
    right_j6: 3.1908623046875
